src/libs/ReportUtils.ts

```diff
@@ -7040,6 +7040,42 @@ function findPolicyExpenseChatByPolicyID(policyID: string): OnyxEntry<Report> {
     return Object.values(allReports ?? {}).find((report) => isPolicyExpenseChat(report) && report?.policyID === policyID);
 }
 
+function getIntegrationIcon(integrationName?: ValueOf<typeof CONST.POLICY.CONNECTIONS.NAME>) {
+    if (integrationName === CONST.POLICY.CONNECTIONS.NAME.XERO) {
+        return XeroSquare;
+    }
+    if (integrationName === CONST.POLICY.CONNECTIONS.NAME.QBO) {
+        return QBOSquare;
+    }
+    return undefined;
+}
+
+function getIntegrationDisplayName(integrationName?: ValueOf<typeof CONST.POLICY.CONNECTIONS.NAME>) {
```

Comment:
This function won't be needed if you apply the changes that I suggested!

Reply:
Can we leave it, but change it to:
```javascript
function getIntegrationDisplayName(connectionName: ConnectionName) {
    return CONST.POLICY.CONNECTIONS.NAME_USER_FRIENDLY[connectionName];
}
```